discovery/seata-discovery-zk/src/test/java/io/seata/discovery/registry/zk/ZookeeperRegisterServiceImplTest.java

```diff
@@ -59,12 +59,6 @@ public void getInstance() {
         Assertions.assertEquals(service1, service);
     }
 
-    @Test
```

Comment:
Why should this test case be deleted? If it must be deleted, why did the previous test case pass?

Reply:
because the test case in `ZookeeperRegisterServiceImplTest` would cause some problem when parallel execution. And the have cover the same code so if delete one of them it would not cause the coverage decrease.